graylog2-web-interface/src/components/lookup-tables/adapters/DSVHTTPAdapterFieldSet.jsx

```diff
@@ -107,4 +103,13 @@ const DSVHTTPAdapterFieldSet = ({ handleFormEvent, validationState, validationMe
   );
 };
 
+DSVHTTPAdapterFieldSet.propTypes = {
+  config: PropTypes.object.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
+  updateConfig: PropTypes.func.isRequired,
```

Comment:
This prop is marked as unused, do we really use it in the component or can we delete it?

Reply:
Fixed